dts/bindings/mtd/jedec,jesd216.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2018 Peter Bigot Consulting, LLC
+# Copyright (c) 2019-2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+# Common properties used by nodes describing serial flash devices that
+# are compatible with the JESD216 Serial Flash Discoverable Parameters
+# specification.
+#
+# This allows encoding the entire BFP block in devicetree to avoid
+# reading at runtime, while still allowing the driver to pull out extra
+# data of interest, such as erase sizes.
+#
+# Alternatively the BFP block can be absent, but critical fields like
+# size can be provided directly, along with the JEDEC ID of to validate
```

Comment:
I guess there is typo, _of to_ validate

Reply:
More a matter of changing the structure of the sentence while writing and not getting it cleaned up.  Thanks; fixed now.